RKAnalysis/python/tools/nanoAOD/BParking_modules.py

```diff
@@ -1,12 +1,12 @@
-
-def SkimCuts(Bdecay,Bcuts):
+def SkimCutsForData(Bdecay,Bcuts):
     BParking_skim_cut = ("Sum$( "+
 #                         "BToKMuMu_l_xy_unc>0 && BToKMuMu_l_xy/BToKMuMu_l_xy_unc>-1 && "+
                          Bdecay+"_fit_pt>{ptmin} && "+
                          Bdecay+"_fit_mass>{mmin} && "+
                          Bdecay+"_fit_mass<{mmax} && "+
                          Bdecay+"_l_xy_unc>0 && "+
-                         Bdecay+"_l_xy/"+Bdecay+"_l_xy_unc>{slxy} && "+
+                         Bdecay+"_l_xy/"+
+                         Bdecay+"_l_xy_unc>{slxy} && "+
```

Comment:
I'm just curious why you re-formatted this bit, was the significance not computed correctly before? or just the formatting is just better now?